salt/modules/win_groupadd.py

```diff
@@ -65,7 +65,7 @@ def _get_group_object(name):
     """
     with salt.utils.winapi.Com():
         nt = win32com.client.Dispatch("AdsNameSpaces")
-        return nt.GetObject("", "WinNT://./" + name + ",group")
+        return nt.GetObject("", "WinNT://./" + str(name) + ",group")
```

Comment:
This would be better as an f-string which would automatically cast name as a string.
```
>>> name = 123
>>> "WinNT://./" + name + ",group"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate str (not "int") to str
>>> f"WinNT://./{name},group"
'WinNT://./123,group'
>>> 
```

Reply:
Got it!